cpp/benchmarks/io/cuio_common.cpp

```diff
@@ -201,3 +201,51 @@ void try_drop_l3_cache()
                            [](auto& cmd) { return exec_cmd(cmd).empty(); }),
                "Failed to execute the drop cache command");
 }
+
+cudf::io::io_type retrieve_io_type_enum(std::string const& io_string)
```

Comment:
nit: `std::string_view` instead of `std::string const&`
similarly at other function as well.

Reply:
```suggestion
cudf::io::io_type retrieve_io_type_enum(std::string_view io_string)
```

Reply:
Thanks! I'll keep that in mind.